qtc_packaging/debian_fremantle/control

```diff
@@ -7,11 +7,11 @@ Build-Depends: debhelper (>= 5), libqt4-dev, libx11-dev
 Standards-Version: 3.7.3
 Homepage: http://maemo.org/packages/view/xbmcremote/
 
-Package: xbmcremote
+Package: kodimote
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libqjson, libqtm-12-contacts
```

Comment:
Hmm - I only removed the `${misc:Depends}` entry, as it will only emit a warning during packaging. The `${shlibs:Depends}` and the other items are of course needed (new line 12 in the diff).
And yes - `libqjson` took me about an hour to figure out. :) `mad-admin xdpkg -i` is only half the truth, as the `.list` and `.shlibs` are not installed...